install/tutorial_12.md

```diff
@@ -183,22 +183,22 @@ Miniconda suffices.
 
 Create if necessary, and activate a Conda environment:
 
-    conda create -n ign-ws
-    conda activate ign-ws
+    conda create -n gz-ws
+    conda activate gz-ws
 
 Install prerequisites:
 
     conda install urdfdom --channel conda-forge
 
-Install Ignition dependencies:
+Install Gazebo dependencies:
 
 You can view lists of dependencies:
 
     conda search libsdformat --channel conda-forge --info
 
 Install dependencies, replacing `<#>` with the desired versions:
 
-    conda install libignition-math<#> libignition-tools<#> --channel conda-forge
+    conda install libignition-math<#> libgz-tools<#> --channel conda-forge
```

Comment:
Can you revert this since for libsdformat12, we'd still need `libignition-*` packages.